CRM/Upgrade/Incremental/php/FiveFortySeven.php

```diff
@@ -83,6 +83,16 @@ public function upgrade_5_47_alpha1($rev): void {
     }
   }
 
+  /**
+   * Upgrade step; adds tasks including 'runSql'.
+   *
+   * @param string $rev
+   *   The version number matching this function name
+   */
+  public function upgrade_5_47_1($rev): void {
```

Comment:
@colemanw I would remove this

Reply:
@seamuslee001 [the recommendation](https://github.com/civicrm/civicrm-core/pull/22893#issuecomment-1060071181) from @totten was to leave it in 5.47.1

Reply:
If the step is safe to re-run, then do move it.

If it's not safe to re-run... then... we don't *have* to move it. We put scary language like https://download.civicrm.org/latest/ in anticipation of these edge-cases.

I think if the time-frame to have affected sites is small (like 1 day of exposure; and only on RC sites), then I don't fell much guilt about doing it the lazy way. But if we're looking at, say, 4 days... then I'd push harder for a safer variant.

Reply:
K I'll move it. We also have to run Regen.sh on this one.